Exporter/src/main/java/uk/ac/ebi/biostd/exporter/service/SubmissionService.java

```diff
@@ -21,11 +22,14 @@
 @Service
 @AllArgsConstructor
 public class SubmissionService {
+    private static final String SEPARATOR = ".";
+    private static final String LIB_FILE_SUFFIX = ".files.json";
 
     private final SubmissionDao submissionDao;
     private final SectionDao sectionDao;
     private final FilesDao filesDao;
     private final LinksDao linksDao;
+    private final GeneralConfiguration config;
```

Comment:
Configuration can be injected as beans but their pupose is only provide beans,by adding as a dependency we are copling it to functional code.